CommitPoolContract/contracts/SinglePlayerCommit.sol

```diff
@@ -58,8 +58,10 @@ contract SinglePlayerCommit is ChainlinkClient, Ownable {
         uint256 stake
     );
     event CommitmentEnded(address committer, bool met, uint256 amountPenalized);
-    event Deposit(address committer, uint256 amount);
-    event Withdrawal(address committer, uint256 amount);
+    event Deposit(address committer, uint256 amount, uint256 committerBalance);
+    event FundsSlashed(address committer, uint256 amount, uint256 slashedBalance);
```

Comment:
do we need a separate funds slashed event in addition to CommittmentEnded (which includes the amountPenalized)? 

or maybe this is here for debugging purposes?

Reply:
There's certainly room for polishing the events. For instance, emitting events also consumes gas and in some cases there are now at least 2 events. So we might only want to do them on the public level, so for owner/consumer functions.

Maybe something like CommitmentEnded(committer, success, amount)?